testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -1263,6 +1263,23 @@ public void testInsertDeletUpdateWithPrestoAndHive()
         });
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL, timeOut = TEST_TIMEOUT)
+    public void testDeletePartitionedTable()
```

Comment:
There already is `testBucketedPartitionedDelete` which you can use, or add `testUnbucketedPartitionedDelete` (per existing convention in the class)

see also https://github.com/trinodb/trino/pull/7299

Reply:
I wanted to have the exact test case from the issue.

Reply:
it's good for development,  but these tests are slow... whatever